pkg/cmd/schedulerplugin/main.go

```diff
@@ -40,6 +40,9 @@ func main() {
 	pluginconf.BuildVersion = version
 	pluginconf.BuildDate = date
 
+	conf.Version = version
+	conf.Date = date
+
```

Comment:
These should replace the plugin config BuildVersion and BuildDate, those should be removed here and from the pluginconf code